src/stravavis/plot_landscape.py

```diff
@@ -0,0 +1,34 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+
+def plot_landscape(df, output_file = 'landscape.png'):
+    
```

Comment:
We need a clean slate:

```suggestion

    # Create a new figure
    plt.figure()

```

Otherwise images are superimposed (`strava-landscape.png`):

![strava-landscape](https://user-images.githubusercontent.com/1324225/152957428-1df1471f-baba-49ba-92be-b222445913b0.png)

(This from the CI: [Summary page](https://github.com/marcusvolz/strava_py/actions/runs/1811007160) > Artifacts > [images](https://github.com/marcusvolz/strava_py/suites/5215903833/artifacts/159694426))